SLURM GPU devices: 0

CondaError: Run 'conda init' before 'conda deactivate'

INFO 08-08 13:37:32 [__init__.py:235] Automatically detected platform cuda.
INFO 08-08 13:37:48 [config.py:1604] Using max model len 1024000
INFO 08-08 13:37:49 [config.py:2434] Chunked prefill is enabled with max_num_batched_tokens=16384.
/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/mistral_common/tokens/tokenizers/tekken.py:337: FutureWarning: The attributed `special_token_policy` is deprecated and will be removed in 1.10.0. Please pass a special token policy explicitly to the relevant methods.
  warnings.warn(
INFO 08-08 13:37:51 [core.py:572] Waiting for init message from front-end.
INFO 08-08 13:37:51 [core.py:71] Initializing a V1 LLM engine (v0.10.0) with config: model='/pfs/work9/workspace/scratch/ul_swv79-pixtral/pixtral-12b/', speculative_config=None, tokenizer='/pfs/work9/workspace/scratch/ul_swv79-pixtral/pixtral-12b/', skip_tokenizer_init=False, tokenizer_mode=mistral, revision=None, override_neuron_config={}, tokenizer_revision=None, trust_remote_code=False, dtype=torch.bfloat16, max_seq_len=1024000, download_dir=None, load_format=auto, tensor_parallel_size=1, pipeline_parallel_size=1, disable_custom_all_reduce=False, quantization=None, enforce_eager=False, kv_cache_dtype=auto,  device_config=cuda, decoding_config=DecodingConfig(backend='auto', disable_fallback=False, disable_any_whitespace=False, disable_additional_properties=False, reasoning_backend=''), observability_config=ObservabilityConfig(show_hidden_metrics_for_version=None, otlp_traces_endpoint=None, collect_detailed_traces=None), seed=0, served_model_name=/pfs/work9/workspace/scratch/ul_swv79-pixtral/pixtral-12b/, num_scheduler_steps=1, multi_step_stream_outputs=True, enable_prefix_caching=True, chunked_prefill_enabled=True, use_async_output_proc=True, pooler_config=None, compilation_config={"level":3,"debug_dump_path":"","cache_dir":"","backend":"","custom_ops":[],"splitting_ops":["vllm.unified_attention","vllm.unified_attention_with_output","vllm.mamba_mixer2"],"use_inductor":true,"compile_sizes":[],"inductor_compile_config":{"enable_auto_functionalized_v2":false},"inductor_passes":{},"use_cudagraph":true,"cudagraph_num_of_warmups":1,"cudagraph_capture_sizes":[512,504,496,488,480,472,464,456,448,440,432,424,416,408,400,392,384,376,368,360,352,344,336,328,320,312,304,296,288,280,272,264,256,248,240,232,224,216,208,200,192,184,176,168,160,152,144,136,128,120,112,104,96,88,80,72,64,56,48,40,32,24,16,8,4,2,1],"cudagraph_copy_inputs":false,"full_cuda_graph":false,"max_capture_size":512,"local_cache_dir":null}
INFO 08-08 13:37:54 [parallel_state.py:1102] rank 0 in world size 1 is assigned as DP rank 0, PP rank 0, TP rank 0, EP rank 0
/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/mistral_common/tokens/tokenizers/tekken.py:337: FutureWarning: The attributed `special_token_policy` is deprecated and will be removed in 1.10.0. Please pass a special token policy explicitly to the relevant methods.
  warnings.warn(
Using a slow image processor as `use_fast` is unset and a slow processor was saved with this model. `use_fast=True` will be the default behavior in v4.52, even if the model was saved with a slow processor. This will result in minor differences in outputs. You'll still be able to use a slow processor with `use_fast=False`.
ERROR 08-08 13:37:57 [core.py:632] EngineCore failed to start.
ERROR 08-08 13:37:57 [core.py:632] Traceback (most recent call last):
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core.py", line 623, in run_engine_core
ERROR 08-08 13:37:57 [core.py:632]     engine_core = EngineCoreProc(*args, **kwargs)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core.py", line 441, in __init__
ERROR 08-08 13:37:57 [core.py:632]     super().__init__(vllm_config, executor_class, log_stats,
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core.py", line 77, in __init__
ERROR 08-08 13:37:57 [core.py:632]     self.model_executor = executor_class(vllm_config)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/executor/executor_base.py", line 53, in __init__
ERROR 08-08 13:37:57 [core.py:632]     self._init_executor()
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/executor/uniproc_executor.py", line 48, in _init_executor
ERROR 08-08 13:37:57 [core.py:632]     self.collective_rpc("init_device")
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/executor/uniproc_executor.py", line 58, in collective_rpc
ERROR 08-08 13:37:57 [core.py:632]     answer = run_method(self.driver_worker, method, args, kwargs)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/utils/__init__.py", line 2985, in run_method
ERROR 08-08 13:37:57 [core.py:632]     return func(*args, **kwargs)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/worker/worker_base.py", line 603, in init_device
ERROR 08-08 13:37:57 [core.py:632]     self.worker.init_device()  # type: ignore
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/worker/gpu_worker.py", line 189, in init_device
ERROR 08-08 13:37:57 [core.py:632]     self.model_runner: GPUModelRunner = GPUModelRunner(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/worker/gpu_model_runner.py", line 146, in __init__
ERROR 08-08 13:37:57 [core.py:632]     encoder_compute_budget, encoder_cache_size = compute_encoder_budget(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/core/encoder_cache_manager.py", line 199, in compute_encoder_budget
ERROR 08-08 13:37:57 [core.py:632]     ) = _compute_encoder_budget_multimodal(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/core/encoder_cache_manager.py", line 229, in _compute_encoder_budget_multimodal
ERROR 08-08 13:37:57 [core.py:632]     .get_max_tokens_per_item_by_nonzero_modality(model_config)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/registry.py", line 158, in get_max_tokens_per_item_by_nonzero_modality
ERROR 08-08 13:37:57 [core.py:632]     self.get_max_tokens_per_item_by_modality(model_config).items()
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/registry.py", line 132, in get_max_tokens_per_item_by_modality
ERROR 08-08 13:37:57 [core.py:632]     return profiler.get_mm_max_tokens(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/profiling.py", line 287, in get_mm_max_tokens
ERROR 08-08 13:37:57 [core.py:632]     mm_inputs = self._get_dummy_mm_inputs(seq_len, mm_counts)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/profiling.py", line 170, in _get_dummy_mm_inputs
ERROR 08-08 13:37:57 [core.py:632]     processor_inputs = factory.get_dummy_processor_inputs(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/profiling.py", line 92, in get_dummy_processor_inputs
ERROR 08-08 13:37:57 [core.py:632]     dummy_text = self.get_dummy_text(mm_counts)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/model_executor/models/llava.py", line 192, in get_dummy_text
ERROR 08-08 13:37:57 [core.py:632]     processor = self.info.get_hf_processor()
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/model_executor/models/llava.py", line 288, in get_hf_processor
ERROR 08-08 13:37:57 [core.py:632]     return self.ctx.get_hf_processor(PixtralProcessor, **kwargs)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/inputs/registry.py", line 138, in get_hf_processor
ERROR 08-08 13:37:57 [core.py:632]     return super().get_hf_processor(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/inputs/registry.py", line 101, in get_hf_processor
ERROR 08-08 13:37:57 [core.py:632]     return cached_processor_from_config(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/transformers_utils/processor.py", line 110, in cached_processor_from_config
ERROR 08-08 13:37:57 [core.py:632]     return cached_get_processor(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/transformers_utils/processor.py", line 72, in get_processor
ERROR 08-08 13:37:57 [core.py:632]     processor = processor_factory.from_pretrained(
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/transformers/processing_utils.py", line 1314, in from_pretrained
ERROR 08-08 13:37:57 [core.py:632]     return cls.from_args_and_dict(args, processor_dict, **kwargs)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/transformers/processing_utils.py", line 1115, in from_args_and_dict
ERROR 08-08 13:37:57 [core.py:632]     processor = cls(*args, **valid_kwargs)
ERROR 08-08 13:37:57 [core.py:632]   File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/transformers/models/pixtral/processing_pixtral.py", line 110, in __init__
ERROR 08-08 13:37:57 [core.py:632]     self.image_token_id = tokenizer.convert_tokens_to_ids(self.image_token)
ERROR 08-08 13:37:57 [core.py:632] AttributeError: 'MistralTokenizer' object has no attribute 'convert_tokens_to_ids'. Did you mean: 'convert_tokens_to_string'?
Process EngineCore_0:
Traceback (most recent call last):
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core.py", line 636, in run_engine_core
    raise e
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core.py", line 623, in run_engine_core
    engine_core = EngineCoreProc(*args, **kwargs)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core.py", line 441, in __init__
    super().__init__(vllm_config, executor_class, log_stats,
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core.py", line 77, in __init__
    self.model_executor = executor_class(vllm_config)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/executor/executor_base.py", line 53, in __init__
    self._init_executor()
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/executor/uniproc_executor.py", line 48, in _init_executor
    self.collective_rpc("init_device")
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/executor/uniproc_executor.py", line 58, in collective_rpc
    answer = run_method(self.driver_worker, method, args, kwargs)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/utils/__init__.py", line 2985, in run_method
    return func(*args, **kwargs)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/worker/worker_base.py", line 603, in init_device
    self.worker.init_device()  # type: ignore
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/worker/gpu_worker.py", line 189, in init_device
    self.model_runner: GPUModelRunner = GPUModelRunner(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/worker/gpu_model_runner.py", line 146, in __init__
    encoder_compute_budget, encoder_cache_size = compute_encoder_budget(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/core/encoder_cache_manager.py", line 199, in compute_encoder_budget
    ) = _compute_encoder_budget_multimodal(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/core/encoder_cache_manager.py", line 229, in _compute_encoder_budget_multimodal
    .get_max_tokens_per_item_by_nonzero_modality(model_config)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/registry.py", line 158, in get_max_tokens_per_item_by_nonzero_modality
    self.get_max_tokens_per_item_by_modality(model_config).items()
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/registry.py", line 132, in get_max_tokens_per_item_by_modality
    return profiler.get_mm_max_tokens(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/profiling.py", line 287, in get_mm_max_tokens
    mm_inputs = self._get_dummy_mm_inputs(seq_len, mm_counts)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/profiling.py", line 170, in _get_dummy_mm_inputs
    processor_inputs = factory.get_dummy_processor_inputs(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/multimodal/profiling.py", line 92, in get_dummy_processor_inputs
    dummy_text = self.get_dummy_text(mm_counts)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/model_executor/models/llava.py", line 192, in get_dummy_text
    processor = self.info.get_hf_processor()
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/model_executor/models/llava.py", line 288, in get_hf_processor
    return self.ctx.get_hf_processor(PixtralProcessor, **kwargs)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/inputs/registry.py", line 138, in get_hf_processor
    return super().get_hf_processor(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/inputs/registry.py", line 101, in get_hf_processor
    return cached_processor_from_config(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/transformers_utils/processor.py", line 110, in cached_processor_from_config
    return cached_get_processor(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/transformers_utils/processor.py", line 72, in get_processor
    processor = processor_factory.from_pretrained(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/transformers/processing_utils.py", line 1314, in from_pretrained
    return cls.from_args_and_dict(args, processor_dict, **kwargs)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/transformers/processing_utils.py", line 1115, in from_args_and_dict
    processor = cls(*args, **valid_kwargs)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/transformers/models/pixtral/processing_pixtral.py", line 110, in __init__
    self.image_token_id = tokenizer.convert_tokens_to_ids(self.image_token)
AttributeError: 'MistralTokenizer' object has no attribute 'convert_tokens_to_ids'. Did you mean: 'convert_tokens_to_string'?
Traceback (most recent call last):
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/Pixtral-Finetune/inference_scripts/test_tokenizer.py", line 3, in <module>
    llm = LLM(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/entrypoints/llm.py", line 273, in __init__
    self.llm_engine = LLMEngine.from_engine_args(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/engine/llm_engine.py", line 497, in from_engine_args
    return engine_cls.from_vllm_config(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/llm_engine.py", line 126, in from_vllm_config
    return cls(vllm_config=vllm_config,
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/llm_engine.py", line 103, in __init__
    self.engine_core = EngineCoreClient.make_client(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core_client.py", line 77, in make_client
    return SyncMPClient(vllm_config, executor_class, log_stats)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core_client.py", line 514, in __init__
    super().__init__(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/core_client.py", line 408, in __init__
    with launch_core_engines(vllm_config, executor_class,
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/contextlib.py", line 142, in __exit__
    next(self.gen)
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/utils.py", line 697, in launch_core_engines
    wait_for_engine_startup(
  File "/pfs/work9/workspace/scratch/ul_swv79-pixtral/conda/pixtral_inference/lib/python3.10/site-packages/vllm/v1/engine/utils.py", line 750, in wait_for_engine_startup
    raise RuntimeError("Engine core initialization failed. "
RuntimeError: Engine core initialization failed. See root cause above. Failed core proc(s): {}

============================= JOB FEEDBACK =============================

NodeName=uc3n082
Job ID: 1094758
Cluster: uc3
User/Group: ul_swv79/ul_student
State: FAILED (exit code 1)
Nodes: 1
Cores per node: 8
CPU Utilized: 00:00:24
CPU Efficiency: 4.92% of 00:08:08 core-walltime
Job Wall-clock time: 00:01:01
Memory Utilized: 1.25 GB
Memory Efficiency: 1.96% of 64.00 GB (64.00 GB/node)
